src/components/plotly/shared/styles/index.js

```diff
@@ -4,14 +4,15 @@ import { setup, styled } from 'goober'
 import plotStyles from './plot'
 import legendStyles from './legend'
 import utilStyles from './util'
+import DivBase from '../div-base'
 
 setup(createElement)
```

Comment:
To be able to work with camel case props in goober, we can use one of the 2 mechanisms forwardProps, or shouldForwardProp:
https://goober.js.org/api/setup/#:~:text=With%20forwardProps-,%23,-The%20forwardProps%20function